### PR TITLE
Update REFERENCE.md

### PR DESCRIPTION
**Pull request by FrRichard**
basic documentation for remote.book, parameters format and events

### CODE REVIEW

File: docs/REFERENCE.md [hunk 1]
```diff
@@ -5,6 +5,7 @@ __(More examples coming soon!)__
 ###In this document:
 
 1. [`Remote` options](REFERENCE.md#remote-options)
+  + [Orderbook remote](REFERENCE.md#remote-orderbook)
```
Comment (clark800) [inline docs/REFERENCE.md]: This probably belongs in a new category called "Subscriptions" (number 4) and could be called "Orderbook Subscription".

**Comment by clark800** [comment]
LGTM after squash

**Comment by FrRichard** [comment]
I'm not extremely familiar with this part of github, do I need to do anything before it can be added to the current documentation ?

**Comment by clark800** [comment]
Yeah, you just have to squash the three commits down to one commit, which you can do with `git rebase -i`.

**Comment by FrRichard** [comment]
ok got it

**Comment by clark800** [comment]
Thanks!

**Pull request merged**